src/core/api/logging_api.cpp

```diff
@@ -41,7 +41,7 @@ using namespace ot;
 
 otLogLevel otLoggingGetLevel(void)
 {
-#if OPENTHREAD_CONFIG_LOG_LEVEL_DYNAMIC_ENABLE && !OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE
+#if OPENTHREAD_CONFIG_LOG_LEVEL_DYNAMIC_ENABLE
     return Instance::Get().GetLogLevel();
```

Comment:
Should we change this as well ?
```suggestion
    return Instance::GetLogLevel();
```

Reply:
There is also another use in `mesh_forwarder.cpp` where we have `GetInstance().GetLogLevel()` and may be better to change it to `Instance::GetLogLevel()`?

Reply:
Thanks, Updated!